tests/test_connector.py

```diff
@@ -594,6 +601,7 @@ async def test_tcp_connector_multiple_hosts_errors(loop: Any) -> None:
     ip4 = "192.168.1.4"
```

Comment:
We need a similar test to make sure it falls back to the ipv4 host but should be quite light as to not reimplement testing the library